### PR TITLE
Use helpers.logError instead of console.error (CU-jcuw85)

### PR DESCRIPTION
**Pull request by trisapeace**
- This will add the timestamp to error output in production

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-jcuw85 add timestamps to server error logs](https://app.clickup.com/t/jcuw85)

**Review by mariocimet — APPROVED** [review]

**Review by schwarrrtz — APPROVED** [review]
looks good.

File: server/db/db.js [hunk 1]
```diff
@@ -15,8 +15,7 @@ const pool = new Pool({
 })
 
 pool.on('error', err => {
-  // eslint-disable-next-line no-console
-  console.error('unexpected database error:', err)
+  helpers.logError('unexpected database error:', err)
```
Comment (trisapeace) [inline server/db/db.js:18]: Hmmm, actually now that I'm looking at this again, I think that the `logError` function only takes one parameter. So I probably need to make this all a single string. This is a reminder for me to check this before merging.

Comment (trisapeace) [reply server/db/db.js:18]: Yup, the function only takes in one argument, the string to output. So I've changed this now to do that.

**Comment by trisapeace** [comment]
Waiting for https://github.com/bravetechnologycoop/brave-alert-lib/pull/8 .
Then I'll upgrade the version of `brave-alert-lib` and the timestamps will only appear once in the logs.

**Review by risatronic — APPROVED** [review]
Looks good to me!

**Review by mariocimet — APPROVED** [review]

**Pull request merged**